src/administration/web/configure-environment.md

```diff
@@ -21,7 +29,49 @@ git push origin :BRANCH-NAME
 > **note**
 > Deleting the Master environment is forbidden.
 
-## Basic Authentication / Restrict access to IPs
+### Outgoing emails
+
+From this tab, you can allow your application to send emails via a SendGrid SMTP proxy. 
+
+![env email](/images/env_email.png)
+
+Changing this setting will temporarily list the environment's status as "Dirty". Once it has re-deployed, it will appear once again as "Active" in your settings.
+
+
+### Search engine visibility
+
+From this tab, you can tell search engines to ignore the site entirely, even if it is publicly visible.
+
+![env search](/images/env_search.png)
+
+
+#### X-Robots-Tag
```

Comment:
Going down to a 4th level is too deep most of the time.  Also, there's only the one sub-item here, which means it shouldn't be a sub-item, no?